backend/src/main/resources/static/aas-registry-openapi.yaml

```diff
@@ -1083,7 +1083,7 @@ components:
           type: string
         text:
           minLength: 0
-          pattern: "^([\\t\\n\\r \\_\\.\\/\\:\\#\\-a-zA-Z0-9]|[\\ud800\\udc00-\\ud800\\udfff]|[\\ud801\\udc00-\\udbfe\\udfff]|[\\udbff\\udc00-\\udbff\\udfff])*+$"
+          pattern: "^([\\t\\n\\r -\ud7ff\ue000-\ufffd]|\\ud800[\\udc00-\\udfff]|[\\ud801-\\udbfe][\\udc00-\\udfff]|\\udbff[\\udc00-\\udfff])*$"
```

Comment:
```suggestion
          pattern: ""^([\\t\\n\\r-\\ud7ff\\ue000-\\ufffd]|\\ud800[\\udc00-\\udfff]|[\\ud801-\\udbfe][\\udc00-\\udfff]|\\udbff[\\udc00-\\udfff])*$""
```

Reply:
see https://github.com/admin-shell-io/aas-specs/issues/362#issuecomment-2061268024

Reply:
fixed. Thanks for the info.